helm/templates/deployment.yaml

```diff
@@ -41,7 +41,9 @@ spec:
             - mountPath: /timecard-restapi-keystore
               name: keystore-volume
           env: {{ toYaml .Values.env.database | nindent 12 }}
+          {{- if eq .Values.environment "dev"}}
```

Comment:
Can we test the value for existence rather than tying things to environments
{{ - if not empty .Values.env.kafka }}
Then in the other environments don't set the kafka config

Reply:
actioned